Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM ubuntu:14.04
+FROM ubuntu:17.10
 MAINTAINER dev@dwolla.com
 USER root
 
-ENV SECURITY_MONKEY_VERSION=v0.8.0
+ENV SECURITY_MONKEY_VERSION=v0.9.1
 
 RUN apt-get update &&\
-    apt-get -y install python-pip python-dev python-psycopg2 libpq-dev git libffi-dev &&\
+    apt-get -y install python-pip python-dev python-psycopg2 libpq-dev git libffi-dev python-virtualenv libssl-dev &&\
     cd /usr/local/src &&\
     git clone --depth 1 --branch $SECURITY_MONKEY_VERSION https://github.com/Netflix/security_monkey.git &&\
     cd security_monkey &&\
+    virtualenv venv &&\
```

Comment:
Security monkey has switched their docs to recommend using virtualenv.  It's probably not as important since we are running in a more isolated environment, but I don't think it hurts anything.  

I wasn't sourcing the activate file for venv correctly.  It is now.  

I also added the WORKDIR command to the bottom of the file which will drop future containers build on this in the virtualenv.  Most other containers will want to end up there and run `./monkey` commands.